determined-ai/determined
# feat: add SearchDetails page (ET-53)

## Ticket

ET-53


## Description




## Test Plan

With Flat Runs feature flag on:
- Open a project details page
- Open the "Searches" tab
- Clicking on the 'ID' or 'Name' of a search should link to the Search Details page
  - Trials tab should display table with trials for the Search
  - Code tab should display code for the Search
  - Notes tab should display editable notes specific to the Search

## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
